site2/docs/how-to-landing.md

```diff
@@ -0,0 +1,16 @@
+---
+Id: how-to-landing
+title: How-to

```

Comment:
Do we need more explanations for `title`? Does `How-to` provide too limited info?

Reply:
We've had several discussions about this. I suggest we get the material posted and iterate corrections as needed.